CHANGELOG.md

```diff
@@ -13,7 +13,18 @@
   config.sidekiq.propagate_traces = false unless Rails.const_defined?('Server')
   ```
 - Only expose `active_storage` keys on span data if `send_default_pii` is on ([#2589](https://github.com/getsentry/sentry-ruby/pull/2589))
-- Add `Sentry.capture_log` ([#2606](https://github.com/getsentry/sentry-ruby/pull/2617))
+- Add new `Sentry.logger` for [Structured Logging](https://develop.sentry.dev/sdk/telemetry/logs/) feature ([#2620](https://github.com/getsentry/sentry-ruby/pull/2620)).
+
+  To enable structured logging you need to turn on the `enable_logs` configuration option:
+  ```ruby
+  Sentry.init do |config|
+    # ... your setup ...
+    config.enable_logs = true
+  end
+  ```
```

Comment:
please also add 2-3 examples here in the changelog for how to use the `capture_log` and `Sentry.logger` APIs - with different levels, with templates/parameters

Reply:
@sl0thentr0py done via c41de52a4d9f244daed9ad44d72c6fdbcdbbcc1f